cs/test/core/StreamTests.cs

```diff
@@ -212,6 +219,54 @@ protected static byte[] MakeSequential(int length)
 
             return buf;
         }
+
+        [Conditional("RELEASE")]
+        private void Stream_PositionLength_NotAccessedOnWriteStructBeginImplementation()
+        {
+            var stream = new NonSeekableStream();
+            var output = new OutputStream(stream, 11);
```

Comment:
Minor: can you used named parameters so that the 11 and 2 have names associated with them?